packages/api/src/models/Line.ts

```diff
@@ -10,11 +10,11 @@ export enum LineType {
 
 export default interface Line {
 	amount?: Money
-	basecampId?: string
+	basecampId?: number
 	compoundedTax?: boolean
 	date?: Date
```

Comment:
I think we should also take this out. It's not used anymore and not documented https://www.freshbooks.com/api/invoices

Reply:
I notice we have the same for Invoice as well, I am guessing we can take that out as well?